lib/node_modules/@stdlib/complex/float64/base/div/test/test.strided.js

```diff
@@ -0,0 +1,1054 @@
+
+		if ( q[ 0 ] === qre[ i ] ) {
+			t.strictEqual( q[ 0 ], qre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - qre[ i ] );
+			tol = EPS * abs( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. real: '+q[0]+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === qim[ i ] ) {
+			t.strictEqual( q[ 1 ], qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - qim[ i ] );
+			tol = EPS * abs( qim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. imag: '+q[1]+'. expected: '+qim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes a complex quotient (tiny positive real components)', function test( t ) {
+	var delta;
+	var tol;
+	var re1;
+	var im1;
+	var re2;
+	var im2;
+	var qre;
+	var qim;
+	var z1;
+	var z2;
+	var i;
+	var q;
+
+	re1 = tinyPositiveRealComponents.re1;
+	im1 = tinyPositiveRealComponents.im1;
+	re2 = tinyPositiveRealComponents.re2;
+	im2 = tinyPositiveRealComponents.im2;
+	qre = tinyPositiveRealComponents.qre;
+	qim = tinyPositiveRealComponents.qim;
+
+	for ( i = 0; i < re1.length; i++ ) {
+		z1 = new Float64Array( [ re1[ i ], im1[ i ] ] );
+		z2 = new Float64Array( [ re2[ i ], im2[ i ] ] );
+		q = cdiv( z1, 1, 0, z2, 1, 0, new Float64Array( 2 ), 1, 0 );
+
+		if ( q[ 0 ] === qre[ i ] ) {
+			t.strictEqual( q[ 0 ], qre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - qre[ i ] );
+			tol = EPS * abs( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. real: '+q[0]+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === qim[ i ] ) {
+			t.strictEqual( q[ 1 ], qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - qim[ i ] );
+			tol = EPS * abs( qim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. imag: '+q[1]+'. expected: '+qim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function handles large and small numbers', function test( t ) {
+	var expected;
+	var z1;
+	var z2;
+	var v;
+
+	z1 = new Float64Array( [ 1.0e308, 5.0e307 ] );
+	z2 = new Float64Array( [ 1.0, 0.5 ] );
+	v = cdiv( z1, 1, 0, z2, 1, 0, new Float64Array( 2 ), 1, 0 );
+	expected = new Float64Array( [ 1.0e308, 0.0 ] );
+
+	t.strictEqual( isAlmostEqualFloat64Array( v, expected ), true, 'returns expected values' );
+
+	z1 = new Float64Array( [ 1.0, 0.5 ] );
+	z2 = new Float64Array( [ 1.0e308, 5.0e307 ] );
+	v = cdiv( z1, 1, 0, z2, 1, 0, new Float64Array( 2 ), 1, 0 );
+	expected = new Float64Array( [ 1.0e-308, 0.0 ] );
+
+	t.strictEqual( isAlmostEqualFloat64Array( v, expected ), true, 'returns expected values' );
```

Comment:
```suggestion
	t.strictEqual( isAlmostEqualFloat64Array( v, expected, maxULPs ), true, 'returns expected values' );
```

Missing an argument. Applies here and elsewhere: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/assert/is-almost-equal-float64array.